apps/client/src/app/export/export.component.html

```diff
@@ -36,6 +36,7 @@ <h1>Export</h1>
             <label>Output Format</label>
             <select class="form-control" [(ngModel)]="options.responseFormat">
               <option value="application/json">JSON</option>
+              <option value="application/msword">MSWORD</option>
```

Comment:
The MWord export options are bundled in under the "IG Publisher Package" tab. "MS Word" should be a separate tab entirely, with (in the future) its own export options. Remove the option from the "select" field, and just create a separate tab for "MS Word". When the user selects the tab, onTabChange() will be called and the options.exportFormat should be changed to (a new enum option) "msword".

Reply:
Completed.